app/views/admin/users/_admin_dashboard.html.slim

```diff
@@ -0,0 +1,58 @@
+.row
+  .col-lg-12
+    .page-header
+      h2.center Admin dashboard
+.row
+  .col-md-6.col-lg-6.col-sm-6
+    .panel.panel-default.panel-white.core-box
+      .panel-body.no-padding
+        .partition.partition-grey.padding-20.core-icon.text-center
+          .glyphicon.glyphicon-question-sign.fa-2x
+        .padding-20.core-content
+          h2
+            = Question.count
+          h4 Questions
+      .panel-footer.clearfix.no-padding.text-center
+        a.col-xs-6.padding-10.partition-light-blue href="/admin/questions/new"
+          .glyphicon.glyphicon-plus
+        a.col-xs-6.padding-10.partition-blue href="/admin/questions"
+          .glyphicon.glyphicon-chevron-right
+  .col-md-6.col-lg-6.col-sm-6
+    .panel.panel-default.panel-white.core-box
+      .panel-body.no-padding
+        .partition.partition-grey.padding-20.core-icon.text-center
+          .glyphicon.glyphicon-th-list
+        .padding-20.core-content
+          h2
+            = Topic.count
+          h4 Topics
+      .panel-footer.clearfix.no-padding.text-center
+        a.col-xs-6.padding-10.partition-light-blue href="/admin/topics/new"
+          .glyphicon.glyphicon-plus
+        a.col-xs-6.padding-10.partition-blue href="/admin/topics" 
+          .glyphicon.glyphicon-chevron-right
+  .col-md-6.col-lg-6.col-sm-6
+    .panel.panel-default.panel-white.core-box
+      .panel-body.no-padding
+        .partition.partition-grey.padding-20.core-icon.text-center
+          .glyphicon.glyphicon-user
+        .padding-20.core-content
+          h2
+            = User.count
+          h4 Users
+      .panel-footer.clearfix.no-padding.text-center
+        a.col-xs-12.padding-10.partition-blue href="/admin/users"
+          .glyphicon.glyphicon-chevron-right
+  .col-md-6.col-lg-6.col-sm-6
+    .panel.panel-default.panel-white.core-box
+      .panel-body.no-padding
+        .partition.partition-grey.padding-20.core-icon.text-center
+          .glyphicon.glyphicon-time
+        .padding-20.core-content
+          h2
+            = User.waiting_approval.count
+          h4 Users waiting approval

```

Comment:
we shouldn't show "UserS" in plural when it's one user
use something from here http://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize